framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/DeploymentScenarioBuilderFactory.java

```diff
@@ -16,22 +16,13 @@
 package org.kie.cloud.api;
 
 import java.util.Iterator;
+import java.util.List;
 import java.util.ServiceLoader;
 
 import org.kie.cloud.api.scenario.builder.WorkbenchWithKieServerScenarioBuilder;
 
 public interface DeploymentScenarioBuilderFactory {
+    String getCloudEnvironmentName();
```

Comment:
cloud environment name vs. CLOUD_API_IMPLEMENTATION constant - lets unify the naming, please